src/rendering/renderable.cpp

```diff
@@ -179,12 +167,16 @@ double Renderable::boundingSphere() const {
     return _boundingSphere;
 }
 
+double Renderable::interactionSphere() const {
+    return _interactionSphere;
+}
+
 SurfacePositionHandle Renderable::calculateSurfacePositionHandle(
                                                  const glm::dvec3& targetModelSpace) const
 {
     const glm::dvec3 directionFromCenterToTarget = glm::normalize(targetModelSpace);
     return {
-        directionFromCenterToTarget * boundingSphere(),
+        directionFromCenterToTarget * _parent->interactionSphere(),
```

Comment:
Changing to the interaction sphere here makes sense, as this function is mainly used for navigation. Although, I think the name of the function becomes a little misguiding, as we are no longer computing a handle to the surface of the renderable (the interaction sphere could theoretically be very different from the actual "surface" of the renderable). 

Don't have a good idea about a better name though, so maybe it's enough to add an explaining comment for the function in the header file? Pointing out that it is a handle to a point on the interaction sphere

Reply:
I added a comment to it'  the SurfacePositionHandle was already documented generally enough:
```
 * Defines the position of an object relative to a surface. The surface is defined as
 * a reference surface together with a height offset from that reference surface.
 *
```